rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -235,21 +229,14 @@ public String getId() {
 
             private boolean isLocked = false;
 
-            private long lockTime = -1;
-
-            private String nodeLocker = null;
-
             private String eventType = null;
 
-            Node(String nodeUrl, String nodeState, String nodeInfo, long timeStamp, String timeStampFormatted,
-                    String nodeProvider, String nodeOwner, String sourceName, String hostName, String vmName,
-                    String description, String defaultJMXUrl, String proactiveJMXUrl, boolean isLocked, long lockTime,
-                    String nodeLocker, String eventType) {
+            Node(String nodeUrl, String nodeState, long timeStamp, String nodeProvider, String nodeOwner,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 13 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)